### DISCUSSION TITLE
默认尺寸下  按钮高度是28px 部分输入框高度是30px  如果放到一行高度不一致，请问这个是bug么 还是就是这么设计得

### DISCUSSION THREAD

**Discussion by senyintgithub** [opentiny/tiny-vue#139] [General]
默认尺寸下  按钮高度是28px 部分输入框高度是30px  如果放到一行高度不一致，请问这个是bug么 还是就是这么设计得

**Comment by kagol** [comment]
> 默认尺寸下  按钮高度是28px 部分输入框高度是30px  如果放到一行高度不一致，请问这个是bug么 还是就是这么设计得

这应该是一个问题
